ledger/ledger-api-tests/infrastructure/BUILD.bazel

```diff
@@ -43,7 +43,7 @@ load(
             "@maven//:org_slf4j_slf4j_api",
         ],
     )
-    for lf_version in lf_version_configuration_versions
+    for lf_version in lf_version_configuration_versions + ["1.14"]
```

Comment:
maybe it's better if we define `lf_supported_versions = lf_version_configuration_versions + ["1.14"]` next to `lf_version_configuration_versions`?